Mathlib/Data/Matrix/Kronecker.lean

```diff
@@ -417,10 +418,11 @@ Prefer the notation `⊗ₖₜ` rather than this definition. -/
 def kroneckerTMul : Matrix l m α → Matrix n p β → Matrix (l × n) (m × p) (α ⊗[R] β) :=
   kroneckerMap (· ⊗ₜ ·)
 
+@[inherit_doc kroneckerTMul]
 scoped[Kronecker] infixl:100 " ⊗ₖₜ " => Matrix.kroneckerMap (· ⊗ₜ ·)
 
-scoped[Kronecker]
-  notation:100 x " ⊗ₖₜ[" R "] " y:100 => Matrix.kroneckerMap (TensorProduct.tmul R) x y
+@[inherit_doc kroneckerTMul]
+scoped[Kronecker] notation:100 x " ⊗ₖₜ[" R "] " y:100 => Matrix.kroneckerMap (TensorProduct.tmul R) x y
```

Comment:
```suggestion
@[inherit_doc kroneckerTMul] scoped[Kronecker] notation:100 x " ⊗ₖₜ[" R "] " y:100 => 
  Matrix.kroneckerMap (TensorProduct.tmul R) x y
```
Odd that it is too long now. Above is a suggestion -- take it or leave it. I don't remember the amount of indentation for line breaks on notation. Returning to what you had before would be ok too.